config/config.yaml

```diff
@@ -21,9 +21,9 @@ ref:
 primers:
   trimming:
     activate: false
-    # path to amplicon file in the format defined by ptrimmer:
-    # https://github.com/DMU-lilab/pTrimmer
-    primers: "path/to/primer-txt"
+    # path to fasta files containg primer sequences
+    primers_fa1: "path/to/primer-fa1"
+    primers_fa2: "path/to/primer-fa2"
```

Comment:
there should be a separate config in .test that runs the workflow without primer trimming

Reply:
How does adding an additional config work as the Snakefile links to the existing one?